### ISSUE TITLE
Harmonize popup/dialogs lookandfeel

### ISSUE THREAD

**Issue by mbernasocchi** [opengisch/QField#634]
We have various different looking popup displays. We should try to unify the UI

# Status

### feature dialog
![Screenshot_20190927-145321_QField Beta 624](https://user-images.githubusercontent.com/233663/65770856-39c25a00-e137-11e9-8be9-1f42b210f5df.jpg)

### Installing
![Screenshot_20190927-145928_QField Beta 624](https://user-images.githubusercontent.com/233663/65771050-a4739580-e137-11e9-9398-2c7483b0ae1a.jpg)

### Wms login
almost good, just the background looks a bit off
![wms_login](https://user-images.githubusercontent.com/28384354/64340665-fddc1f00-cfe6-11e9-8afd-59c2dfe58b48.gif)

### Sidebar
some issues in spacing that might already be fixed by #628 
![Screenshot_20190927-145308_QField Beta 624](https://user-images.githubusercontent.com/233663/65770852-37600000-e137-11e9-8c17-8da257979774.jpg)

### Changelog
maybe green title bar with white text like the feature dialog
![Screenshot_20190927-145025_QField Beta 624](https://user-images.githubusercontent.com/233663/65770832-3038f200-e137-11e9-8452-95d7987cada9.jpg)

### file selector
possibe to add a margin around?
![Screenshot_20190927-145045_QField Beta 624](https://user-images.githubusercontent.com/233663/65770843-33cc7900-e137-11e9-9aec-3b068d516993.jpg)

### Welcome screen
![Screenshot_20190927-145035_QField Beta 624](https://user-images.githubusercontent.com/233663/65770839-3202b580-e137-11e9-98c9-2e5bb04dfe49.jpg)

## longpress on a legend item
![Screenshot_20190927-150529_QField Beta 624 (1)](https://user-images.githubusercontent.com/233663/65771552-bf92d500-e138-11e9-8620-bdc5c636c6b3.jpg)

# suggestion
try to unify the look and feel of the various dialogues similar to the Feature dialog.

I think since most the things are using hard corners we can stick with even though Material surfaces have a rectangular shape by default, with 4dp rounded corners. The unpacking files would then be the only with rounded corners but I think it can be neglected being visible only once.

**Comment by m-kuhn** [comment]
Good idea to harmonize these.

* I would be in favor of removing the border on the feature dialog
* I'd not change put menus and dialogs into the same bucket. Can we move them to a different issue?
* For the welcome dialog, we should have less text and more graphics
* The unpacking dialog has rectangular shape here, so it seems to be system dependent. +1 to not put any focus on that.

**Comment by mbernasocchi** [comment]
> Good idea to harmonize these.
> 
> * I would be in favor of removing the border on the feature dialog
open to whatever looks best.
> * I'd not change put menus and dialogs into the same bucket. Can we move them to a different issue?
are the longpresses also menues? could you open a new issue? 
> * For the welcome dialog, we should have less text and more graphics
absolutely, nobody reads :)
> * The unpacking dialog has rectangular shape here, so it seems to be system dependent. +1 to not put any focus on that.
+1

**Comment by m-kuhn** [comment]
> are the longpresses also menues? could you open a new issue?

Yes, they are.
Well, I'm not sure what's wrong with them, so I have a hard time formulating the issue myself :)

**Comment by mbernasocchi** [comment]
moved the menus into #635

**Comment by nirvn** [comment]
@mbernasocchi , @m-kuhn , harmony:
![Screenshot from 2019-09-28 15-14-05](https://user-images.githubusercontent.com/1728657/65813678-c4bc5680-e202-11e9-9a02-01119633f33c.png)

The realm label is missing, since that's just a test popup.

**Comment by mbernasocchi** [comment]
I like that a lot, just the 1px green padding top right is wrong :)
